src/components/Cart.js

```diff
@@ -0,0 +1,19 @@
+import CartItem from "./CartItem";
+
+const Cart = ({cartArray, handleRemoveFromCart}) => {
+    return (
+        <ul className="cartList wrapper">
+            <h2>Your Cart</h2>
+            {
+                cartArray.map((cartItemInfo) => {
+                    return <CartItem key={cartItemInfo.key} productInfo={cartItemInfo} handleRemoveFromCart={handleRemoveFromCart} />
```

Comment:
I noticed there are components for handle add to cart/handle remove from cart - I wonder if it's possible for you to compact the file. Since the button is only used once throughout the code (but displayed multiple times due to being in a list), would it be possible for you to just put the two buttons and the functions in the CartItem component nested <li> inside the .map() return and then just display <CartItem /> in the Cart.js ? That way, the buttons will display for each item, but you'd have two less components and you wouldn't deal with props as much! 
I think alternatively, there is also the option of you exporting them as named imports (export without "default" keyword)??  <https://github.com/HackerYou/bootcamp-notes/blob/main/applied-javascript/making-our-code-more-modular.md>

Reply:
Thank you for this suggestion! I went ahead and removed the buttons as components and nested them inside the <li>'s. It made my code a lot simpler/cleaner!